### PR TITLE
Add function to get the subscription usage of the organization

### PR DESCRIPTION
**Pull request by shomix**
linear issue: https://linear.app/spheron/issue/SPH-1516/add-getusageorganizationid-method-the-storage-sdk

PR Includes:
- added a method to get the token scope
- added a method to get the subscription usage of the organization

### CODE REVIEW

**Review by dusanstanis — APPROVED** [review]

File: packages/storage/src/spheron-api/interfaces.ts [hunk 1]
```diff
@@ -120,4 +120,40 @@ interface Deployment {
   updatedAt: Date;
 }
 
+export interface UsageWithLimits {
+  usedBandwidth?: number; // Bytes
+  usedBuildExecution?: number; // Seconds
+  usedConcurrentBuild?: number;
+  usedStorageArweave?: number; // Bytes
+  usedStorageSkynet?: number; // Bytes
```
Comment (shomix) [inline packages/storage/src/spheron-api/interfaces.ts:128]: @rekpero should we remove the usedStorageSkynet from the interface?

Comment (rekpero) [reply packages/storage/src/spheron-api/interfaces.ts:128]: yes, we should

**Review by rekpero — DISMISSED** [review]

File: packages/storage/src/spheron-api/interfaces.ts (hunk 1 above)
Comment (shomix) [reply packages/storage/src/spheron-api/interfaces.ts]: Removed.

**Review by rekpero — DISMISSED** [review]

**Review by dusanstanis — DISMISSED** [review]

**Review by dusanstanis — APPROVED** [review]

File: packages/storage/src/spheron-api/interfaces.ts [hunk 2]
```diff
@@ -120,4 +120,43 @@ interface Deployment {
...
+  domainsLimit?: number;
+  hnsDomainsLimit?: number;
+  ensDomainsLimit?: number;
+  environmentsLimit?: number;
+  clusterAktLimit?: number;
+  clusterBuildExecutionLimit?: number;
+  passwordProtectionLimit?: number;
+}
+
+export interface UsageWithLimitsWithSkynet extends UsageWithLimits {
```
Comment (rekpero) [inline packages/storage/src/spheron-api/interfaces.ts:157]: why is this interface required?

**Review by rekpero — CHANGES_REQUESTED** [review]

File: packages/storage/src/spheron-api/interfaces.ts (hunk 2 above)
Comment (shomix) [reply packages/storage/src/spheron-api/interfaces.ts:157]: This is what the API is returning

Comment (shomix) [reply packages/storage/src/spheron-api/interfaces.ts:157]: But we are removing this properties in the `SpheronClinet` object, and just returning the `UsageWithLimits` interface.

**Review by rekpero — APPROVED** [review]

**Pull request merged**